packages/build/README.md

```diff
@@ -36,10 +36,10 @@ To use `@loopback/build` for your package:
     "clean": "lb-clean loopback-grpc*.tgz dist*",
     "prepare": "npm run build && npm run build:apidocs",
     "pretest": "npm run build:current",
-    "acceptance": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/acceptance/**/*.js'",
-    "integration": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/integration/**/*.js'",
-    "test": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js' 'DIST/test/integration/**/*.js' 'DIST/test/acceptance/**/*.js'",
-    "unit": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js'",
+    "acceptance": "lb-mocha --opts ../../test/mocha.opts 'DIST/test/acceptance/**/*.js'",
+    "integration": "lb-mocha --opts ../../test/mocha.opts 'DIST/test/integration/**/*.js'",
+    "test": "lb-mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js' 'DIST/test/integration/**/*.js' 'DIST/test/acceptance/**/*.js'",
```

Comment:
Ditto. I believe `lb-mocha` is already adding `--opts` to CLI arguments.

Please make sure these scripts actually work when executed from `packages/build` directory. I would expect to see `../bin/run-mocha` here, because it is my understanding that `node_module/.bin/lb-mocha` is created only when `@loopback/build` is installed as a (dev)dependency, which is not the case in `@loopback/build` itself.

Reply:
Also `--opts ../../test/mocha/opts` looks suspicious to me, isn't this path pointing to monorepo-level `test` folder that no longer exists? I think we should not be needed `--opts` at all, since `lb-mocha` (`run-mocha`) is already adding them.